src/r_defs.h

```diff
@@ -125,7 +125,7 @@ typedef struct sector_s
   // or ceilingpic, because the rest of Doom needs to know which is sky
   // and which isn't, etc.
 
- int sky;
+ int floorsky, ceilingsky;
```

Comment:
Hmm, why do we need `floorsky` and `ceilingsky`?

Reply:
Because you can transfer either the floor or the ceiling flat to a sky texture and leave the respective other one untouched. This is what happens in the latest KEX's Doom2/MAP20 at the map exit.

Reply:
Understood, thank you.